src/run/run.c

```diff
@@ -102,6 +103,7 @@ static int help(void) {
                "     --description=TEXT           Description for unit\n"
                "     --slice=SLICE                Run in the specified slice\n"
                "     --slice-inherit              Inherit the slice\n"
+               "     --expand-environment=BOOL    Control server-side expansion of envvars\n"
```

Comment:
there's no shame in using a second line and spelling out "environment variables"

Reply:
(i mean, usually the identifiers we use are shorer than the help texts. Here it's the reverse, right now ;-)

Reply:
Thinking about this, actually "server-side" needs to be dropped in the later patch that expands this to `--scope`. I think I forgot to do that.

Reply:
Yeah. So I amended the last patch to drop "server-side" here.